README.md

```diff
@@ -1014,6 +1015,24 @@ public StepProcessor createProcessor(final StepProcessorBuilder processorBuilder
 }
 ```
 
+### Workflow parameters
+
+Workflow parameters allow you to configure the settings of your Workflow steps without specifying the value. You can create a Workflow parameter and assign it to any of the Workflow step setting that support the Workflow parameter's datatype. 
```

Comment:
should be workflow step property instead of workflow step setting

Reply:
Thank you @chungkhenhah, I have updated the Workflow parameters section :)